helm/charts/aleph/templates/api.yaml

```diff
@@ -149,6 +149,12 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
+{{- if .Values.api.serviceAnnotations }}
```

Comment:
If you specify default value it will always be there, you don't need to check that it exists.